### PR TITLE
LRQA-69761 SubmitFormWithMultipleFieldsAndViewEntriesViaDescriptiveDisplay Test Fix

### PR DESCRIPTION
**Pull request by leticiachiu**
Related Issue: https://issues.liferay.com/browse/LRQA-69761

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by leticiachiu** [comment]
Hey @natocesarrego! This macro is failing this test for days now on CI, even though it passes locally (see more [here](https://testray.liferay.com/home/-/testray/case_results/1559147233?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D1559145683%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc)). Today I asked to Diogo if I could take it out and he said yes, but you think that this step is necessary? I don't know why this macro isn't working since we have other test (Forms#CanBeViewedAfterDeletingAField) that uses it and is passing on CI normally :thinking:

**Comment by natocesarrego** [comment]
@leticiachiu, let's talk about it today after all daily meeting. Can you please remind me about it?

Maybe, we need to add a wait before try to change the display style to Descriptive. :thinking:

**Comment by natocesarrego** [comment]
@leticiachiu, I just tried to reproduce the problem locally and I think that I could do it, please see the video below:
https://user-images.githubusercontent.com/6729287/132524113-4db1f09b-daa2-4be8-b07a-17a48394576a.mp4

From what I could see, the click on the display style icon doesn't show the menu items, hence the next click fails. There is a macro named `WaitForPageLoad();` inside the macro `LexiconEntry.changeDisplayStyle`, however I think that sometimes it's not enough.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 1]
```diff
@@ -2281,7 +2281,11 @@ definition {
 			category = "Content &amp; Data",
 			portlet = "Forms");
 
-		LexiconEntry.changeDisplayStyle(displayStyle = "list");
+		Click.clickNoMouseOverNoWaitForVisible(locator1 = "Icon#CURRENT_DISPLAY_STYLE");
+
+		Click.clickNoMouseOverNoWaitForVisible(
+			key_menuItem = StringUtil.upperCaseFirstLetter("list"),
+			locator1 = "MenuItem#ANY_MENU_ITEM");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:2288]: @leticiachiu, from what I could understand from this change, your proposal is perform the clicks without waiting for the elements load on the screen, am I right?

If so, won't it go in the opposite direction of what we've discussed yesterday? Can you please clarify?

Thanks

Comment (leticiachiu) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:2288]: Ooh, I misunderstood this macro.. Sorry :dizzy:

**Comment by leticiachiu** [comment]
@natocesarrego Now I changed the old macro but it doesn't affects other tests (I checked)

File: portal-web/test/functional/com/liferay/portalweb/macros/LexiconEntry.macro [hunk 2]
```diff
@@ -3,7 +3,7 @@ definition {
 	macro changeDisplayStyle {
 		WaitForPageLoad();
 
-		Click(locator1 = "Icon#CURRENT_DISPLAY_STYLE");
+		Click.mouseOverClick(locator1 = "Icon#CURRENT_DISPLAY_STYLE");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/macros/LexiconEntry.macro:6]: Good job, @leticiachiu!

**Review by natocesarrego — COMMENTED** [review]
@leticiachiu, with the proposed fix, can we remove this test from quarantine? How will we know if this test has passed in CI environment?

**Comment by natocesarrego** [comment]
ci:forward

**Comment by liferay-forms** [comment]
ci:test:relevant

**Pull request closed without merging**